streampark-common/src/main/scala/org/apache/streampark/common/util/FileUtils.scala

```diff
@@ -58,6 +64,27 @@ object FileUtils {
       }) == "504B0304"
   }
 
+  def isPythonFileType(contentType: String, input: InputStream): Boolean = {
```

Comment:
def isPythonFileType(contentType: String, input: InputStream): Boolean = {
    if (StringUtils.isBlank(contentType) || input == null) {
      throw new RuntimeException("The contentType or inputStream can not be null")
    }

    contentType.contains("text/x-python") && getMimeType(input).equals(MediaType.TEXT_PLAIN.toString)
  }